### PR TITLE
add nobg code snippet

### PR DESCRIPTION
**Pull request by not-lain**
this pr adds code snippet for nobg: https://github.com/feyninc/nobg
a library i spearheaded its development which you can find one model under https://huggingface.co/feyninc/FeyNobg where it achieves SOTA performance across multiple benchmarks.

---

> [!NOTE]
> **Low Risk**
> Metadata and static code snippets only; no runtime, auth, or inference pipeline changes.
> 
> **Overview**
> Registers the **nobg** background-removal library on Hub model pages and shows copy-paste usage for models tagged with `library_name: nobg`.
> 
> Adds a `nobg` entry to `MODEL_LIBRARIES_UI_ELEMENTS` (label, GitHub repo link, snippet hook; not added to the models filter). Introduces a `nobg(model)` snippet generator with `pip install nobg` plus an example that loads `AutoModel` / `AutoProcessor` from the repo id, runs inference, post-processes alpha matting, and saves a cutout PNG.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit 7a312a52200b8c05bba9e1fe6058ae247881500a. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

File: packages/tasks/src/model-libraries-snippets.ts [hunk 1]
```diff
@@ -2266,6 +2187,26 @@ from models.birefnet import BiRefNet
 model = BiRefNet.from_pretrained("${model.id}")`,
 ];
 
+export const nobg = (model: ModelData): string[] => [
+	`# !pip install nobg
+
+import torch
+from loadimg import load_img
```
Comment (not-lain) [inline packages/tasks/src/model-libraries-snippets.ts:2194]: this comes pre-installed with the library

File: packages/tasks/src/model-libraries-snippets.ts [hunk 2]
```diff
@@ -2266,6 +2187,26 @@ from models.birefnet import BiRefNet
 model = BiRefNet.from_pretrained("${model.id}")`,
 ];
 
+export const nobg = (model: ModelData): string[] => [
+	`# !pip install nobg
```
Comment (pcuenca) [inline packages/tasks/src/model-libraries-snippets.ts]: Note you can return an array with multiple steps, you can use the first item for installation. See, for example, the entry for "chronos-forecasting" in this file, which renders like this:

Image

**Review by pcuenca — COMMENTED** [review]
Nice!

File: packages/tasks/src/model-libraries-snippets.ts (hunk 2 above)
Comment (not-lain) [reply packages/tasks/src/model-libraries-snippets.ts]: @pcuenca thanks a lot for the follow up, updated the code snippet to match your suggestion

**Review by pcuenca — APPROVED** [review]

**Review by Wauplin — APPROVED** [review]
🔥
